docs/quickstart.md

```diff
@@ -15,24 +15,14 @@
 
 ## Download the Automation Code
 
-Head to the [release](https://github.com/ocp-power-automation/ocp4-upi-powervs/releases) page and download the latest stable release.
-Extract the release bundle on your system.
+You'll need to use git to clone the deployment code when working off the master branch
 
-You can also use `curl` or `wget` to download the stable release code as shown below.
 ```
```

Comment:
Shouldn't we have both the options (curl and wget) documented  ?

Reply:
@bpradipt This is to align the docs as per master branch. IMO `git clone` is enough as we mentioned Git as a per-requisite.

Reply:
That might be an oversight. Will approve this to go through.  In subsequent PRs will add the mention of downloading via curl or wget